### PR TITLE
Remove why files and clarify in README

### LINKED ISSUE #56: Replace WHY_*.md with WORKFLOW.md
Describing how I like to work would be better than trying to explain why I use particular tools.

A single file summarising how I like to work would be better than partitioning how I work into different places - this is important as I use tools together and don't use my text editor, the terminal or the browser by themselves, so why would I talk about them in isolation.

I should write this up so that when I am working intensely on a project, I have thought out workflow rules to follow as well as general principles to apply when I use new tooling.

Thoughts to address:

##### Automate as much as possible.
+ Record your answers so you will only improve on your how you think and you won't have to re answer/rethink it.
+ Reviewing code mistakes which can be caught by a linter is a waste of time - automation is the answer.
+ Fixing typos/addressing simple grammar mistakes is a waste of time, there is tooling for this.
+ Testing - what to test - thoughts on coverage.
##### Importance of documentation
How to write documentation (https://hackernoon.com/why-senior-devs-write-dumb-code-and-how-to-spot-a-junior-from-a-mile-away-27fa263b101a)
Nothing should ever need to be explained.
##### Workflow in the terminal
(Importance of a linter, but not to lint on save. Why I don't like the noise of a directory tree on my screen? Thoughts on autocompletion. Thoughts on syntax highlighting).
##### Importance of not rushing
(https://github.com/elm/projects/blob/master/roadmap.md#what-is-the-timeline)

### LINKED ISSUE #82: Group Progams to install section
Currently we have one big list of `Programs to install` we should consider breaking these up into different sections if this list will grow any bigger.

### LINKED ISSUE #83: Add Sketch
Add [Sketch](https://sketchapp.com/) to `Applications to download` section and write why I chose to use this app.

### PR DESCRIPTION
**Pull request by samhstn**
+ remove _WHY_ files (fixes #56).
+ change Chrome to Brave browser.
+ add Sketch (fixes #83).
+ group programs and applications (fixes #82).
+ ensure all images are a user-images.githubusercontent.com link.
+ ensure consistent img tags.
+ change Spectacle for Rectangle as it's no longer being maintained
(https://github.com/eczarny/spectacle#important-note).
+ add Qbserve and Freedom.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,472 +1,302 @@
...
++ In the `Utility` menu, select `Disk Utility`.
++ Click `Erase` on all `Internal` drives (where permitted).
++ Once this has completed, click `Done`, then close the window to get back to the original menu.
++ Click `Reinstall macOS`, then `Continue`.
 
-+ [**Homebrew**](https://github.com/samhstn/My-Config/tree/master#homebrew)
+## Applications and programs to install
 
-+ [**Git**](https://github.com/samhstn/My-Config/tree/master#git)
++ [**Brave**](#brave) - For a web browser with unmatched speed, security and privacy.
```
Comment (nelsonic) [inline README.md:20]: 👍

File: README.md [hunk 2]
```diff
@@ -1,472 +1,302 @@
...
-+ [**Vim**](https://github.com/samhstn/My-Config/tree/master#vim)
++ Back up what you need.
++ Boot your mac in recovery mode (Restart while holding `Cmd + R`).
++ In the `Utility` menu, select `Disk Utility`.
++ Click `Erase` on all `Internal` drives (where permitted).
++ Once this has completed, click `Done`, then close the window to get back to the original menu.
++ Click `Reinstall macOS`, then `Continue`.
 
-+ [**Homebrew**](https://github.com/samhstn/My-Config/tree/master#homebrew)
+## Applications and programs to install
```
Comment (nelsonic) [inline README.md:18]: great list of tools @samhstn! a few I haven't tried. adding to list for Saturday experimentation. 👍

File: README.md [hunk 3]
```diff
@@ -1,472 +1,302 @@
...
+In `System preferences > Trackpad`:
++ tick: `Tap to click`.
 
-+ [Vimium](https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb?hl=en) - Allows for browsing the web without leaving your keyboard
+In `System preferences > Keyboard`:
++ Key Repeat: `Fast`est.
++ Delay Until Repeat: `Short`est.
 
-<br><br>
+Hide the dock by right clicking it and selecting: `Turn hiding on`.
```
Comment (nelsonic) [inline README.md:57]: Keyboard shortcut: `Command + Option + D` AKA `[ ⌘ ] + [ Alt ⌥ ] + [ D ]` 😉

File: README.md [hunk 4]
```diff
@@ -1,472 +1,302 @@
...
 
-<img src="http://core0.staticworld.net/images/article/2013/03/splash-100030679-large.jpg" align="right" height="80" />
+In `Security & Privacy > FileVault`:
++ `Turn on FileVault` and set a recovery key.
 
-+ [Lastpass](https://www.lastpass.com/) - For managing all your passwords
+(I like to ignore the recovery key as I never store anything worth keeping on my machine).
```
Comment (nelsonic) [inline README.md:62]: Worth explaining that you use cloud services for all important data. 💭

File: README.md [hunk 5]
```diff
@@ -1,472 +1,302 @@
...
+### [Zsh](https://ohmyz.sh/)
 
-```bash
-brew install python3
-pip3 install --upgrade pip setuptools wheel
-```
+<img src="https://user-images.githubusercontent.com/15983736/75923752-a3206480-5e5d-11ea-9460-e16190f6333c.jpg" alt="oh_my_zsh" align="right" width="150" />
 
-----
+The modern shell supported by an [awesome community](http://ohmyz.sh/).
```
Comment (nelsonic) [inline README.md:198]: `zsh` is now the default shell on Mac: https://support.apple.com/en-us/HT208050 💭 
https://www.theverge.com/2019/6/4/18651872/apple-macos-catalina-zsh-bash-shell-replacement-features

File: README.md [hunk 6]
```diff
@@ -1,472 +1,302 @@
...
 ```bash
-brew install heroku/brew/heroku
-```
-
-----
-
-#### Grip
-
-If looking for a gui markdown previewer, look at using [haroopad](http://pad.haroopress.com)
+curl -SLs https://raw.githubusercontent.com/samhstn/my-config/master/sams-theme.zsh-theme \
```
Comment (nelsonic) [inline README.md:209]: 👍

**Review by nelsonic — APPROVED** [review]
@samhstn looks great! Always good to remove stale items. ♻️ 👍

File: README.md (hunk 5 above)
Comment (samhstn) [reply README.md:198]: Wow, didn't know that!

**Pull request merged**